flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -2546,7 +2557,9 @@ void takeSnapshot() throws Exception {
 
 			// create hard links of living files in the snapshot path
 			try (Checkpoint checkpoint = Checkpoint.create(stateBackend.db)) {
-				checkpoint.createCheckpoint(localBackupDirectory.getDirectory().getPath());
+				LOG.info("Checkpoint path is {}.", localBackupDirectory.getDirectory().getPath());
```

Comment:
I think this should not be logged on info level, it can potentially spam the log file. We could log this on debug or trace.